### PR TITLE
Home v2

### PR DESCRIPTION
**Pull request by nuclearkittens**
+ description of app for users who are not logged in displayed on the home page
+ general fixes and refactoring

### CODE REVIEW

File: src/components/Home.jsx [hunk 1]
```diff
@@ -1,208 +1,169 @@
...
-    const isRestaurantUser = restaurantId !== null;
+    let username, isRestaurantUser;
+    if (userSession) {
+        username = userSession.username;
+        isRestaurantUser = userSession.restaurantId !== null;
+    }
 
-    const deleteMealButtonId = (index) => `delete-meal-button-${index}`;
-    const exportMealQrButtonId = (index) => `export-meal-qr-button-${index}`;
+    if (!userSession) {
```
Comment (Robomarti) [inline src/components/Home.jsx:96]: is this not unreachable, because on line 56 the exact same thing is checked?

File: src/components/MealDeletion.jsx
Comment (Robomarti) [inline src/components/MealDeletion.jsx:1]: MealDeletion is a bit shite name for a component that also handles exporting QR codes and maybe in the future edits meal information too...

**Review by Robomarti — APPROVED** [review]
pretty nice

File: src/components/MealDeletion.jsx
Comment (nuclearkittens) [reply src/components/MealDeletion.jsx:1]: yeah, named it when it only had the deletion. renaming is a good idea

**Pull request merged**